plugins/extra/p2js/componentConfigs/P2BodyConfig.ts

```diff
@@ -24,6 +25,7 @@ export default class P2BodyConfig extends SupCore.Data.Base.ComponentConfig {
     shape: { type: "enum", items: [ "box", "circle" ], mutable: true },
     width: { type: "number", min: 0, mutable: true },
     height: { type: "number", min: 0, mutable: true },
+    angle: { type: "number", min: -360, max: 360, mutable: true },
```

Comment:
Sure, I can change it if you want. I looked at Cannon Body and it uses -360;360 so I though I would do the same. Let me know what you think is best.